source/common/http/access_log/access_log_impl.cc

```diff
@@ -99,7 +99,7 @@ bool RuntimeFilter::evaluate(const RequestInfo&, const HeaderMap& request_header
   const HeaderEntry* uuid = request_header.RequestId();
   uint16_t sampled_value;
   if (uuid && UuidUtils::uuidModBy(uuid->value().c_str(), sampled_value, 100)) {
-    uint64_t runtime_value = std::min(runtime_.snapshot().getInteger(runtime_key_, 0), 100UL);
+    auto runtime_value = std::min<uint64_t>(runtime_.snapshot().getInteger(runtime_key_, 0), 100);
```

Comment:
can we keep uint64_t as the var type (prefer to not use auto outside of for loops for clarity)